dts/dts.cmake

```diff
@@ -54,8 +55,7 @@ if(CONFIG_HAS_DTS)
     -isystem ${PROJECT_SOURCE_DIR}/dts/${ARCH}
     -isystem ${PROJECT_SOURCE_DIR}/dts
     -include ${AUTOCONF_H}
-    ${DTC_INCLUDE_FLAG_FOR_SOURCE}  # include the DTS source
-    ${DTC_INCLUDE_FLAG_FOR_OVERLAY} # Possibly include an overlay after the source
+    ${DTC_INCLUDE_FLAG_FOR_DTS}  # include the DTS source and overlays
```

Comment:
I'm sure there's a reason, but why did these get merged into a single var?

Reply:
To avoid having to convert from a list of files into an `-include <file>` in 2 places. After all, they are exactly the same.